README.md

```diff
@@ -352,6 +352,7 @@ Please take a quick gander at the [contribution guidelines](https://github.com/a
 * [micro](https://github.com/micro/micro) - A pluggable microservice toolkit and distributed systems platform.
 * [NATS](https://github.com/nats-io/gnatsd) - A lightweight, high performance messaging system for microservices, IoT, and cloud native systems.
 * [raft](https://github.com/hashicorp/raft) - Golang implementation of the Raft consensus protocol, by HashiCorp.
+* [raft](https://github.com/coreos/etcd/tree/master/raft) - Go implementation of the Raft consensus protocol, by CoreOS.
```

Comment:
Consider changing the link to https://github.com/coreos/etcd/tree/master/raft#readme, so it's a little more clear what one is looking at. As is, you just see a subdirectory with a bunch of files, which can be confusing if you don't realize there's a README at the bottom.